src/frontend/apps/desk/src/api/fetchApi.ts

```diff
@@ -1,4 +1,4 @@
-import { useAuthStore } from '@/features/';
+import { useAuthStore } from '@/features/auth';
```

Comment:
Alright, okay, I buy it. Then the root problem comes importing multiple modules in `/features/index.ts` that create circular dependencies no? If import from `import ... from '@/features/` is gonna never be used, let's remove it, to avoid create confusion

Reply:
please feel free to ignore my comment if it makes no sense